src/nvim/testdir/shared.vim

```diff
@@ -246,3 +246,7 @@ func! Screenline(lnum)
   let line = join(chars, '')
   return matchstr(line, '^.\{-}\ze\s*$')
 endfunc
+
+func CanRunGui()
```

Comment:
Future vim patches don't use this outside of gui and terminal tests but it's harmless to keep it.